pkgs/tools/misc/dateutils/default.nix

```diff
@@ -0,0 +1,19 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  version = "0.4.1";
+	name = "dateutils-${version}";
+
+  src =fetchurl {
+	url = "https://bitbucket.org/hroptatyr/dateutils/downloads/${name}.tar.xz";
+	sha256 = "0y2jsmvilljbid14lzmk3kgvasn4h7hr6y3wwbr3lkgwfn4y9k3c";
+  };
+
+  meta = with stdenv.lib; {
+    description = "Virtual lighttable and darkroom for photographers";
```

Comment:
Please also fix the description, and any other metadata that may have been copypasted